data/scripts/libsoletta.sym

```diff
@@ -0,0 +1,867 @@
+        sol_util_move_file;
+        sol_util_read_file;
+        sol_util_vread_file;
+        sol_util_vwrite_file;
+        sol_util_write_file;
+        sol_util_write_file_slice;
+
+        sol_util_base16_decode;
+        sol_util_base16_encode;
+        sol_util_base64_calculate_decoded_len;
+        sol_util_base64_decode;
+        sol_util_base64_encode;
+        sol_util_double_eq;
+        sol_util_int32_mul;
+        sol_util_int64_mul;
+        sol_util_replace_str_from_slice_if_changed;
+        sol_util_replace_str_if_changed;
+        sol_util_size_add;
+        sol_util_size_mul;
+        sol_util_size_sub;
+        sol_util_ssize_mul;
+        sol_util_strerror;
+        sol_util_strftime;
+        sol_util_strtod_n;
+        sol_util_strtol_n;
+        sol_util_strtoul_n;
+        sol_util_timespec_get_current;
+        sol_util_timespec_get_realtime;
+        sol_util_uint16_bytes_swap;
+        sol_util_uint32_bytes_swap;
+        sol_util_uint32_mul;
+        sol_util_uint64_add;
+        sol_util_uint64_bytes_swap;
+        sol_util_uint64_mul;
+        sol_util_unescape_quotes;
+        sol_util_unicode_code_from_utf8;
+        sol_util_utf8_from_unicode_code;
+        sol_util_uuid_bytes_from_string;
+        sol_util_uuid_gen;
+        sol_util_uuid_str_is_valid;
+        sol_util_uuid_string_from_bytes;
+
+        sol_str_table_entry_lookup;
+        sol_str_table_int64_entry_lookup;
+        sol_str_table_int64_lookup_fallback;
+        sol_str_table_lookup_fallback;
+        sol_str_table_ptr_entry_lookup;
+        sol_str_table_ptr_lookup_fallback;
+
+        sol_flow_parse_buffer;
+        sol_flow_parse_buffer_metatype;
+        sol_flow_parse_string;
+        sol_flow_parse_string_metatype;
+        sol_flow_parser_del;
+        sol_flow_parser_new;
+
+        sol_json_array_get_at_index;
+        sol_json_calculate_escaped_string_len;
+        sol_json_double_to_str;
+        sol_json_escape_string;
+        sol_json_get_value_by_path;
+        sol_json_is_valid_type;
+        sol_json_load_memdesc;
+        sol_json_object_get_value_by_key;
+        sol_json_path_array_get_segment_index;
+        sol_json_path_get_next_segment;
+        sol_json_path_scanner_init;
+        sol_json_scanner_get_dict_pair;
+        sol_json_scanner_next;
+        sol_json_scanner_skip;
+        sol_json_serialize_bool;
+        sol_json_serialize_double;
+        sol_json_serialize_int32;
+        sol_json_serialize_int64;
+        sol_json_serialize_memdesc;
+        sol_json_serialize_string;
+        sol_json_serialize_uint32;
+        sol_json_serialize_uint64;
+        sol_json_token_get_double;
+        sol_json_token_get_int64;
+        sol_json_token_get_uint64;
+        sol_json_token_get_unescaped_string;
+        sol_json_token_get_unescaped_string_copy;
+
+        sol_update_cancel;
+        sol_update_check;
+        sol_update_fetch;
+        sol_update_get_progress;
+        sol_update_install;
```

Comment:
sol_update here and up in this file. My suggestion is to sort ALL, case SENSITIVE (so `SOL_` comes separated from `sol_`) and do a blank line when the module change (`sol_XXX` and `sol_YYY`).

This will make things easier to maintain